### PR TITLE
Readme and Documentation update

### PR DESCRIPTION
**Pull request by ismaelfaro**
Readme and Doc update
## Description

We add more details and update the Readme and Documentation.

## Types of changes

- [ x ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ x ] I have updated the documentation accordingly.
- [ x ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ x ] All new and existing tests passed.

### CODE REVIEW

File: doc/install.rst [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+    - download the tutorials: https://github.com/QISKit/qiskit-tutorial/archive/master.zip
+    - uncompress the zip file
+    - move the content into a new "tutorials" folder in the QISKit folder
+
+.. code:: sh
+
+    .\make run-tutorials
+    
+
+FAQ
```
Comment (atilag) [inline doc/install.rst]: This more a Troubleshooting section than a FAQ, isn't it?

File: doc/install.rst [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+===
+
+If you upgrade the dependencies and get the error below, try the fix
```
Comment (atilag) [inline doc/install.rst]: Which is the "error below"? Seems like the actual error is missing here

File: README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+The Quantum Information Software Kit (**QISKit** for short) is a software development kit (SDK) for working with [OpenQASM](https://github.com/QISKit/qiskit-openqasm) and the [IBM Quantum Experience (QX)](https://quantumexperience.ng.bluemix.net/). 
+
+Use **QISKit** to create quantum computing programs, compile them, and execute them on one of several backends (online Real Quantum Processors, online Simulators, and local Simulators). For the online backends, QISKit uses our [python API client](https://github.com/QISKit/qiskit-api-py) to connect to the IBM Quantum Experience. 
+
+**We use GitHub issues for tracking requests and bugs. Please see the** [IBM Q Experience community](https://quantumexperience.ng.bluemix.net/qx/community) **for questions and discussion.** **If you'd like to contribute to QISKit, please take a look at our** [contribution guidelines](CONTRIBUTING.rst).
+
+Links to Sections:
+
+* [Installation](#installation-and-setup)
+* [Creating your first Quantum Program](#getting-started)
```
[the remark this answers is not in the document]
Comment (ismaelfaro) [reply README.md]: yes, sorry, we renamed this elements in the last moment.

File: doc/install.rst (hunk 1 above)
Comment (ismaelfaro) [reply doc/install.rst]: in principle we remove the FAQ section.

**Comment by ewinston** [comment]
@ismaelfaro I tried creating a pull request to your fork. Let me know if you don't see it.

**Pull request merged**